guides/common/modules/proc_synchronizing-templates-with-an-existing-repository.adoc

```diff
@@ -1,7 +1,7 @@
 [id="synchronizing-templates-with-an-existing-repository_{context}"]
 = Synchronizing Templates with an Existing Repository
 
-You can use existing repositories to import and export templates from your {ProjectServer}.
+Use this procedure to synchronize templates between your {ProjectServer} and an existing repository.
```

Comment:
Can you/are you meant to sync both ways? If it's one way from source > Foreman, then maybe reword from "between" to "to". I am unsure how this is meant to be used; it's more of a question.

Reply:
See if the new wording makes more sense.

Reply:
If it's both ways, then your first wording was totally OK for me.

Reply:
reworded back.